ntbs-service/Models/Entities/Alerts/DataQualityTreatmentOutcome36.cs

```diff
@@ -17,7 +17,9 @@ public class DataQualityTreatmentOutcome36 : Alert
             n => TreatmentOutcomesHelper.IsTreatmentOutcomeMissingAtXYears(n, 3);
 
         public static readonly Func<Notification, bool> NotificationQualifies =
-            n => NotificationInQualifyingDateRangeExpression.Compile()(n) && NotificationInRangeQualifies(n);
+            n => n.NotificationStatus != NotificationStatus.Closed
+                 && NotificationInQualifyingDateRangeExpression.Compile()(n)
+                 && NotificationInRangeQualifies(n);
```

Comment:
There's clearly a bit of duplication across these classes. Do you think it's worth trying to commonise it?

Reply:
I've changed this to `n.NotificationStatus == NotificationStatus.Notified` based on the requirements Nancy mentioned at standup. I can't think of a good way of commonising this, but I agree it would be a nice thing to do. Do you have any suggestions?

Reply:
I think I could have been a bit more clear here - I was referring to the fact that `DataQualityTreatmentOutcome12`, `DataQualityTreatmentOutcome24` and `DataQualityTreatmentOutcome36` are very similar classes. However, I'm happy to leave this as-is, as it would be quite a moderate gain at this stage.

Reply:
Ah I see what you mean. This would actually be relatively hard to commonise, because the alert service assumes that an alert of each time is present at most once for a record. This requires us to keep the three types, so you might hope to make a `DataQualityTreatmentOutcome` base class with all the logic, and then three little child classes to just feed the year in. Given the way the class is being used the year number would need to be a static (to be called by static methods in the children), but also overridable (to be set for each different class to a different year). Given a field can't be static and overridable at the same time in C#, I think the way OO is implemented in C# means that we're not going to be able to commonise this nicely.